src/main/webapp/view/video/video.js

```diff
@@ -44,8 +44,8 @@ function onPlayerReady(event) {
   window.startVideoSyncTimer();
 }
 
-/** Seeks video to {@code currentTime}. */
-function seekVideo(currentTime) {
+/** Seeks video to {@code currentTimeMs}. */
```

Comment:
here too for @code?

Reply:
I'm going to wait for #222 to be merged so we are consistent everywhere.